scripts/windows_install_mingw64.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+
+echo "starting mingw64 installation"
+
+Invoke-WebRequest -UseBasicParsing -uri "https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe" -OutFile msys2.exe ; 
+if (!$?) { throw 'cmdfail' }
+
+.\msys2.exe -y -oC:\ ;
+Remove-Item msys2.exe ;
+
+
+function msys() { C:\msys64\usr\bin\bash.exe @('-lc') + @Args; } ; 
+msys ' ' ; 
+msys 'pacman --noconfirm -Syuu' ; 
+msys 'pacman --noconfirm -Scc' ; 
+
+echo 'installing packages' ; 
+msys 'pacman --noconfirm --needed -S base-devel \
+    mingw-w64-x86_64-toolchain \
```

Comment:
I successfully built and ran tests with
pacman -S git bison flex make ucrt64/mingw-w64-ucrt-x86_64-perl ucrt64/mingw-w64-ucrt-x86_64-gcc ucrt64/mingw-w64-ucrt-x86_64-zlib diffutils

I think installing ucrt64/mingw-w64-ucrt-x86_64-ccache (and the configuring it in the test) it would also be a good idea.

Reply:
Besides the package selection, it might also be useful to try to use the ucrt64 versions as I did above, instead of the non-ucrt version (those link against more msys / mingw libraries).

Reply:
switched to ucrt4 libs